tests/test_middleware_interface.py

```diff
@@ -68,30 +68,37 @@ def fake_file_data(filename, dimensions, instrument):
     return data_id, file_data
 
 
+# The short name of the instrument used in the test repo.
+instname = "DECam"
+
+
 class MiddlewareInterfaceTest(unittest.TestCase):
     """Test the MiddlewareInterface class with faked data.
     """
     def setUp(self):
         data_dir = os.path.join(os.path.abspath(os.path.dirname(__file__)), "data")
         central_repo = os.path.join(data_dir, "central_repo")
         central_butler = Butler(central_repo,
-                                instrument="LSSTCam",
+                                instrument=instname,
```

Comment:
Should this commit ("Use DECam in tests") be merged with the one that added `central_repo`?

Reply:
Could be? I was trying to be more specific in my commits as you wanted, and this one was mostly atomic.

Reply:
I think this one is subatomic, if the previous `Butler` call wouldn't actually have worked.